content/500-platform/10-about.mdx

```diff
@@ -42,6 +42,33 @@ In each workspace, you can:
 - manage billing, i.e. select a [subscription plan](https://www.prisma.io/pricing?utm_source=docs&utm_medium=platform-docs), configure payment methods, or view the invoice history.
 - view the usage of your enabled PDP products across all projects in that workspace.
 - invite other users to collaborate in the workspace.
+- access the [Optimize](/optimize) dashboard to measure query performance and receive AI-powered recommendations.
+
+#### Optimize
+
+Each workspace in your [Prisma Data Platform account](https://console.prisma.io/optimize) has an associated Optimize project.
+
+##### Accessing the Optimize dashboard
+
+To access the Optimize dashboard in your desired workspace:
+
+1. Click the **Optimize** tab on the left panel.
+2. Click the **Launch Optimize** button.
+
+##### Generating an Optimize API key
+
+To obtain the Optimize API key:
+
+1. Navigate to the workspace where you want to use Optimize.
+2. Ensure that Optimize is launched. If it isn't, click the **Launch Optimize** button.
+3. If it isn't launched, spot the text in the Optimize dashboard that says, "Start by creating your first API key __here__ and start using Optimize," and click the __here__ link.
+4. Click your profile name in the top right corner of the navbar.
+5. Select **Settings**.
+6. Click **Create API key**.
+7. Enter a name for the API key in the **Name** field, then click **Create**.
+8. Copy the API key and store it securely. Finally, click the **I've stored it securely** button.
```

Comment:
maybe link to the .env file suggestion we have? Or just mention that here? "this will be used in your projects .env file via the "WHAT_EVER_VAR_IT_IS"